src/routes/safe/components/Apps/hooks/useSafeAppManifest.ts

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { FETCH_STATUS } from 'src/utils/requests'
+import { getAppInfoFromUrl, getEmptySafeApp } from '../utils'
+import { SafeApp } from '../types'
+
+type UseSafeAppManifestReturnType = {
+  safeApp: SafeApp
+  isLoading: boolean
+}
+
+const useSafeAppManifest = (appUrl: string): UseSafeAppManifestReturnType => {
```

Comment:
It returns a safe app and not the manifest, so I suggest renaming it to `useSafeAppFromManifest`

Reply:
Changed